modules/rosetta/src/main/scala/org/tessellation/rosetta/api/dag/schema.scala

```diff
@@ -1,17 +1,11 @@
-package org.tessellation.rosetta.server.model.dag
-
-import org.tessellation.rosetta.server.model.dag.enums.ChainObjectStatus
+package org.tessellation.rosetta.api.generated.dag
```

Comment:
why these classes are in generated package, if the comment clearly says it's not auto generated?

Reply:
I forgot to update the package name. They have been updated to `package org.tessellation.rosetta.api.dag`.